3d/colored_stl_exporter.py

```diff
@@ -187,13 +192,20 @@ def get_transformed_file_path(original_path):
     @staticmethod
     def parse_openscad_color(color):
         match = RGB_COLOR_REGEX.search(color)
-        if not match:
-            raise ValueError('Failed to parse color. Must be in [<r>, <g>, <b>] format. {}'.format(color))
-        return [
-            float(match.group('r')),
-            float(match.group('g')),
-            float(match.group('b')),
-        ]
+        if match:
+            return [
+                float(match.group('r')),
+                float(match.group('g')),
+                float(match.group('b')),
+            ]
+        if '"' in color and webcolors:
+            try:
+                c = webcolors.name_to_rgb(color[1:-1]) # skip the ""
```

Comment:
nit: what do you think about using regex for this, so that it's more similar to the RGB_COLOR_REGEX usage above it, and then also can assert that the value looks something like `".*"` rather than just checking if a `"` exists anywhere in the string and then attempting to parse it as a webcolor assuming the first and last characters are `"`?